mod/core/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ mkdir -p /var/bigbluebutton/deleted
 mkdir -p /var/bigbluebutton/unpublished
 
 # -- fix directory permissions
-RUN chown bigbluebutton:bigbluebutton /var/bigbluebutton
+RUN chown -R bigbluebutton:bigbluebutton /var/bigbluebutton
```

Comment:
It still does not work.

As this is no longer in a Dockerfile, we need to remove `RUN`.
```suggestion
chown -R bigbluebutton:bigbluebutton /var/bigbluebutton
```

Reply:
May I suggest to add a `set -e` to the script such that such errors are not silently ignored?